installscripts/jazz-terraform-unix-noinstances/vpc_subnet.tf

```diff
@@ -12,10 +12,23 @@ data "aws_vpc" "vpc_data" {
   count = "${var.dockerizedJenkins}"
   id = "${var.autovpc == 1 ? join(" ", aws_vpc.vpc_for_ecs.*.id) : var.existing_vpc_ecs }"
 }
+
+data "external" "instance_ip" {
+  program = ["bash", "-c", "echo \"{\\\"ip\\\" : \\\"$(curl -s  checkip.amazonaws.com)\\\"}\""]
```

Comment:
This won't work at all unless the staging box is an Amazon EC2 instance (something we're trying to get away from assuming is the case in code)

Why does anything need to know the IP of the staging box? Is this another scenario1 requirement?

Reply:
Actually the above code returns the public IP address of either the local machine or the instance. Since ECS containers are restricted to public, we need the installer machine or the local machine's public IP so that it can access the ECS ALBs.

Reply:
Ah! Good man. My bad, thought it was EC2-specific.